controllers/xmlrpc/src/test/java/orca/controllers/mock/MockOrcaConnectionFactory.java

```diff
@@ -48,24 +48,12 @@ public MockOrcaConnectionFactory(String url, String user, String password, ID id
          * @throws Exception
          */
     public synchronized IOrcaServiceManager getServiceManager() throws Exception {
-        // obtain a proxy to the container.
```

Comment:
These are probably left by Jeff when he was originally writing it. There's some historical value here :-)